recipes/extinction/meta.yaml

```diff
@@ -1,46 +0,0 @@
-{% set name = "extinction" %}
```

Comment:
Please don't remove any recipes already included in `staged-recipes`. This should no longer be necessary.

xref: https://github.com/conda-forge/staged-recipes/blob/fb94154aa6a307ed08f7aa206cd0c9ddfa0ef51a/README.md#11-why-are-there-recipes-already-in-the-recipes-directory-should-i-do-something-about-it